### ISSUE TITLE
Broken baseurl on rawgit-prod

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/courses#2]
The fetched url on `rawgit-prod` appears to be broken.

<hr>

[Source](https://github.com/StoDevX/courses/blob/master/rawgit-prod.html#L13)

[Offending url](https://cdn.rawgit.com/StoDevX/course-data/21030278d4a246e63aa5ad246c90fa1e822e3f2f/info.json)

A working [example](https://github.com/StoDevX/courses/blob/master/rawgit-dev.html#L13) of `fetchJson` using `baseURL` on `rawgit-dev`

```js
const baseUrl = 'https://cdn.rawgit.com/StoDevX/course-data/21030278d4a246e63aa5ad246c90fa1e822e3f2f/'
fetchJson(`${baseUrl}/info.json`)
```

**Comment by hawkrives** [comment]
Just delete the two rawgit files please. I'm happy with just using github itself.

**Comment by drewvolz** [comment]
Closed by #4
